.github/labeler.yml

```diff
@@ -19,6 +19,10 @@
       - all-globs-to-any-file:
         - 'frontend/**'
         - '!frontend/viewer/**'
-        - '!frontend/platform.bible-extension/**'
         - '!frontend/pnpm-lock.yaml'
         - '!frontend/pnpm-workspace.yaml'
+
+"📖 Platform.Bible extension":
```

Comment:
I think we'll need to create a label with this name in order for this to work, But I want to sort out the name before we bother with that.

Reply:
yeah that's right I used that because it's a library. How about this:📙(orange book) which looks like a bible if you squint?